src/fzf.rs

```diff
@@ -32,7 +32,7 @@ impl Fzf {
                 "--select-1",
             ]);
             if cfg!(unix) {
-                command.arg("--preview=ls -p {2..}");
+                command.arg(r"--preview=\command -p ls -p {2..}");
```

Comment:
`fzf` uses shell by default, this change gives me this output:
```
fish: Unknown command: \command
fish:
\command -p ls -p '/home'
^
```
Considering that fzf uses whatever shell the user happens to have (which could be something odd like nushell), I think the only option are to just use a bare `ls` and hope that does something sensible, or to fully qualify this as `/bin/ls -p`